azerothcore/azerothcore-wotlk
# (DB/Gameobject_addon) Fixed position for Ironforge Library plaques

## Changes Proposed:
-  Added missing data to `gameobject_addon` table
-  Fixed positions for the plaques, now they are looking upwards


## Issues Addressed:
- Closes #4627 
- Closes https://github.com/chromiecraft/chromiecraft/issues/100



## How to Test the Changes:

1. `.go xyz -4605.23 -1242.93 503.38 0 3.80`
2. Run around the library and check the plaques


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
